lib/members_list_page.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+require_relative 'spanish_congress_page'
+
+class MembersListPage < SpanishCongressPage
```

Comment:
A 'ScrapedPage' with no fields is a bit of a red flag. The first two of these look like they should be fields to me (and the `next_page_link` a private method).